app/src/main/kotlin/com/wire/android/ui/home/messagecomposer/MessageComposerInput.kt

```diff
@@ -258,13 +258,15 @@ private fun InputContent(
                     UsersTypingIndicatorForConversation(conversationId = conversationId)
                 }
             }
-            MessageSendActions(
-                onSendButtonClicked = onSendButtonClicked,
-                sendButtonEnabled = canSendMessage,
-                selfDeletionTimer = viewModel.state(),
-                onChangeSelfDeletionClicked = onChangeSelfDeletionClicked,
-                modifier = Modifier.padding(end = dimensions().spacing8x)
-            )
+            if (canSendMessage || showOptions) {
```

Comment:
please remove this condition it was removed before to show the send button even when the message composer is collapsed

Reply:
resolved in 1:1